File: ePozoriste.Model/Sala.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ePozoriste.Model
{
    public class Sala
    {
        public int SalaId { get; set; }
        public string Naziv { get; set; }
        public int BrSjedista { get; set; }
        public int BrRedova { get; set; }
        public int BrSjedistaPoRedu { get; set; }

        public int? PozoristeId { get; set; }
        public virtual Pozoriste? Pozoriste { get; set; }

        public override string ToString()
        {
            return Naziv;
        }
    }
}

File: epozoriste_mobile/lib/models/sala.dart
import 'package:epozoriste_mobile/models/models.dart';
import 'package:json_annotation/json_annotation.dart';

part 'sala.g.dart';

@JsonSerializable()
class Sala {
  int salaId;
  String naziv;
  int brSjedista;
  int brRedova;
  int brSjedistaPoRedu;
  int? pozoristeId;
  Pozoriste? pozoriste;

  Sala(
      {required this.salaId,
      required this.naziv,
      required this.brSjedista,
      required this.brRedova,
      required this.brSjedistaPoRedu,
      this.pozoristeId,
      this.pozoriste});

  factory Sala.fromJson(Map<String, dynamic> json) => _$SalaFromJson(json);
  Map<String, dynamic> toJson() => _$SalaToJson(this);
}
